extending-valtimo/document/search-fields.md

```diff
@@ -0,0 +1,110 @@
+# Search fields
+
+Search fields add the ability to filter cases. Each search field adds the ability to filter on a specific field.
+This page describes how the search fields can be created, read, updated and deleted.
+
+## Prerequisites
+Only administrators can configure search fields, so the creation, update and deletion of search fields can only be done by administrators.
+
+All logged-in users can retrieve all available search fields using the GET endpoint.
+
+## Creating search fields
+
+There are two methods available to create a search field:
+
+1) Using the endpoint `POST /api/v1/document-search/{documentDefinitionName}/fields` and providing the following request body:
```

Comment:
I somewhat against documenting endpoints like this. It would be better if we could somehow include the swagger documentations the docs.valtimo.nl.

But for this PR its fine I guess

Reply:
This should be a different story in my opinion. This way has been used multiple times in the documentation already